docs/products/postgresql/howto/pg-long-running-queries.rst

```diff
@@ -0,0 +1,87 @@
+Detect and terminate long-running queries
+=========================================
+
+Aiven does not terminate any customer queries even if they run indefinitely, but long-running queries can cause issues by locking resources and therefore preventing database maintenance tasks.
+
+To identify and terminate such long-running queries, you can do it from either:
+
+* `Aiven Console <http://console.aiven.io>`_
+* `PostgreSQL® shell <https://www.postgresql.org/docs/current/app-psql.html>`_ (``psql``)
+
+
+Terminate long running queries from the Aiven console
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In the `Aiven Console <http://console.aiven.io/>`_, you can go to the **Current Queries** tab for your service.
+
+.. image:: /images/products/postgresql/pg-long-running-queries.png
+    :alt: PostgreSQL® service overview tab in Aiven's console
+
+
+On the Current Queries page, you can check the query duration and click **Terminate** to stop any long-running queries.
+
+
+Detect and terminate long running queries with ``psql``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can :doc:`login to your service <./connect-psql>`  by running on the terminal ``psql <service_uri>``.  Once connected, you can call the following function on the ``psql`` shell to terminate a query manually::
+
+    SELECT pg_terminate_backend(pid);
+
+
+You can learn more about the ``pg_terminate_backend()`` function from the `official documentation <https://pgpedia.info/p/pg_terminate_backend.html>`_.
+
+You can then use the following query to monitor currently running queries::
+
+    SELECT * FROM pg_stat_activity WHERE state <> 'idle';
+
+
+Client applications can use the ``statement_timeout`` session variable to voluntarily request the server to automatically cancel any query using the current connection that runs over a specified length of time. For example, the following would cancel any query that runs for more 15 seconds automatically::
+
+    SET statement_timeout = 15000
+
+
+You may check the `client connection defaults <https://www.postgresql.org/docs/current/runtime-config-client.html>`_ documentation for more information on the available session variables.
+
+
+Common error
```

Comment:
Can we make the error code also visible in the title?

Reply:
I'm not sure what the exact error for this would be based on the article.  I'm guess it might be 2203A based on this [error-code appendix](https://www.postgresql.org/docs/current/errcodes-appendix.html), but it's not exactly visible when the error is returned after running the command?